client/apps/shipping-settings/view-wrapper.js

```diff
@@ -66,34 +66,42 @@ class LabelSettingsWrapper extends Component {
 	render() {
 		const { carrier, carrierAccounts, liveRatesTypes, subscriptions, isSaving, translate } = this.props;
 
+		// eslint-disable-next-line no-undef
+		const isWCShippingActive = wcConnectData.is_wcshipping_active;
+
 		if ( ! carrier ) {
 			return (
 				<div>
-					<GlobalNotices id="notices" notices={ notices.list } />
-					<MigratorSettings />
-					<LabelSettings onChange={ this.onChange } />
-					<Packages onChange={ this.onChange } />
-					<LiveRatesCarriersList carrierIds={ liveRatesTypes } />
-					<CarrierAccounts accounts={ carrierAccounts } />
-					<SubscriptionsUsage subscriptions={ subscriptions } />
+					<GlobalNotices id="notices" notices={ notices.list }/>
+					<MigratorSettings/>
+					{ isWCShippingActive && ( <div>
+						<h2>
+							<a href="admin.php?page=wc-settings&tab=shipping&section=woocommerce-shipping-settings">{ translate( 'WooCommerce Shipping' ) }</a> &gt; { translate( 'Live rates' ) }
+						</h2>
+					</div> ) }
```

Comment:
Heading with breadcrumbs (headcrumbs?)

Reply:
Got cha. For user to be able to access or edit packages, it needs to have this complementary PR https://github.com/woocommerce/woocommerce-shipping/pull/748 right? This `view-wrapper.js` changes here is to allow `http://localhost/wp-admin/admin.php?page=wc-settings&tab=shipping&section=woocommerce-services-settings` to load otherwise the breadcrumbs points to the shipping one: 
![image](https://github.com/user-attachments/assets/c3b49725-efc5-4878-9741-ae293be6e9ec)

This PR make the settings page available if accessed via the URL directly; whereas https://github.com/woocommerce/woocommerce-shipping/pull/748 allows the user to access this via a button. This makes sense to me. 👍

Reply:
Actually, they can also access the packages list from WC > Settings > Shipping > Shipping zones > USPS (WooCommerce Shipping) > Edit, and there's a link to package manager there pointing to ☝️ this page. But as a bonus, I'm adding a link to the WCShip settings page in the PR you mentioned so its easier to find.